doc/source/user_guide/ug_implementation.rst

```diff
@@ -684,18 +687,65 @@ layers and the ice thickness distribution defined by ``kcatbound`` = 0.
 Restart information for some tracers is also included in the netCDF restart
 files.
 
-Three namelist variables control model initialization, ``ice_ic``, ``runtype``,
-and ``restart``, as described in :ref:`tab-ic`. It is possible to do an
-initial run from a file **filename** in two ways: (1) set runtype =
-‘initial’, restart = true and ice\_ic = **filename**, or (2) runtype =
-‘continue’ and pointer\_file = **./restart/ice.restart\_file** where
-**./restart/ice.restart\_file** contains the line
-“./restart/[filename]". The first option is convenient when repeatedly
-starting from a given file when subsequent restart files have been
-written. With this arrangement, the tracer restart flags can be set to
-true or false, depending on whether the tracer restart data exist. With
-the second option, tracer restart flags are set to ‘continue’ for all
-active tracers.
+Three namelist variables generally control model initialization, ``runtype``,
+``ice_ic``, and ``use_restart_time``.  The valid values for ``runtype``
+are ``initial`` or ``continue``.  When ``runtype`` = `continue`, the
+initial condition is defined by the rpointer file, ``use_restart_time``
```

Comment:
I'm not sure I like the name "rpointer". I would prefer something like "ice pointer file" or something of sorts... and maybe add a "(see below)" because it's the first time the name is introduced.

Reply:
rpointer is a term in CESM.  I have changed those to "pointer".  thanks for catching that.